hudi-common/src/main/java/org/apache/hudi/io/storage/HoodieAvroFileReader.java

```diff
@@ -48,7 +50,11 @@ protected ClosableIterator<IndexedRecord> getIndexedRecordIterator(Schema reader
     return getIndexedRecordIterator(readerSchema, readerSchema);
   }
 
-  public abstract ClosableIterator<IndexedRecord> getIndexedRecordIterator(Schema readerSchema, Schema requestedSchema) throws IOException;
+  public ClosableIterator<IndexedRecord> getIndexedRecordIterator(Schema readerSchema, Schema requestedSchema) throws IOException {
```

Comment:
nit: should this be final so that it should not be overriden?

Reply:
We have overrides today for all of these in HoodieAvroParquetReader